.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,34 @@
-*Issue #, if available:*
+<!--
+Please make sure to read the Pull Request Guidelines:
 
-*Description of changes:*
+https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#pull-requests
+
+-->
+
+
+#### Description of changes
+<!--
+Thank you for your Pull Request! Please provide a description above and review
+the requirements below.
+
+-->
+
+
+#### Issue #, if available
+
+
+
+#### Description of how you validated changes
+
+
+
+#### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] PR description included
+- [ ] `yarn test` passes
+- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
+- [ ] Relevant documentation is changed or added
```

Comment:
I think we should ask for Relevant Docs PR as we dont want the docs changes to go before the code change, and CLI changes  need a release where as docs is immediate after its merged

Reply:
updated checkbox item and made note under `Issue #` section